packages/synapse-react-client/src/components/CitationPopover/CitationPopover.test.tsx

```diff
@@ -0,0 +1,194 @@
+import { act, cleanup, render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { UseQueryResult } from '@tanstack/react-query'
+import CitationPopover from './CitationPopover'
+import { useCitation } from './useCitation'
+
+jest.mock('./useCitation', () => ({
+  useCitation: jest.fn(),
+}))
+
+const mockUseCitation = useCitation as jest.MockedFunction<typeof useCitation>
+
+const openPopover = async () => {
+  const button = screen.getByTestId('CiteAsButton')
+  await act(async () => {
+    await userEvent.click(button)
+  })
```

Comment:
No need to wrap in `act` when using `userEvent`
```suggestion
  await userEvent.click(button)
```

Reply:
ah I missed this where userEvents internally wrap these actions with wrap!